libbcachefs.h

```diff
@@ -23,6 +23,16 @@ struct {
 };
 };
 
+#define CHECK_OPT_LOCAL(x) (bool)((x & 0x2)>>1)
+#define CHECK_OPT_VALUE(x) (bool)(x & 0x1)
+
+typedef enum {
+	global_false = 0,
+	global_true = 1,
+	local_false = 2,
+	local_true = 3,
+} opt_bool_ext;
```

Comment:
Also please see my comment on issue #33. I'm not clear on what behaviour you're trying to implement.

Reply:
I reverted the commit, but applied a discard=false at the start of every new group, which is less confusing (I guess).  It's not the same as --nodiscard, but for most users it will be the expected behaviour